image/base/Dockerfile

```diff
@@ -130,7 +129,6 @@ ADD install-jemalloc /tmp/install-jemalloc
 RUN /tmp/install-jemalloc
 
 ADD install-redis /tmp/install-redis
```

Comment:
This can be moved too.

Reply:
We'd need a way to add it to the dev image as well when we build that, as that also needs to be able to run install-redis in a different context.

My concern with copying is that any changes to install-redis could get out of sync in an update.